doc/nrf/protocols/matter/index.rst

```diff
@@ -23,9 +23,9 @@ For a full list of |NCS| and Matter versions, view the following table:
    +--------------------------+-----------------------------------------------------+------------------------+
    | nRF Connect SDK version  | Matter specification version                        | Matter SDK version     |
    +==========================+=====================================================+========================+
-   | v2.8.99 (latest)         | :ref:`1.4.0 <ug_matter_overview_dev_model_support>` | 1.4.0.0                |
+   | |release|                | :ref:`1.4.0 <ug_matter_overview_dev_model_support>` | 1.4.0.0                |
```

Comment:
Can we also have v2.9.99 (latest) in the table in this PR?

Reply:
This is normally done in the post-release doc update PR, since v2.9.99 (latest) is not required in the 2.9 release branch.

Reply:
Yeah, that will be done post release.